repository/a.json

```diff
@@ -2306,6 +2306,18 @@
 				}
 			]
 		},
+		{
+			"name": "AutoPEP8_2020",
+			"details": "https://github.com/thep0y/SublimeAutoPEP8",
+			"labels": ["linting", "formatting"],
+			"readme": "https://raw.githubusercontent.com/thep0y/SublimeAutoPEP8/master/Readme.md",
+			"releases": [
+				{
+					"sublime_text": ">=3000",
+					"tags": true
```

Comment:
I noticed by the way that your code has a constant for the version number, but it is behind vs. the latest tag.

Reply:
I will fix it.